### ISSUE TITLE
A Suggestion for 2D pose detection 

### ISSUE THREAD

**Issue by hujb48** [fabro66/GAST-Net-3DPoseEstimation#46]
Hi, It is a great work, and here I used the mediaPipe to instead the Yolov3+HrNet for testing the model in the wild video, the effect is pretty good with quicker speed in processing each frame from 13fps to about 45-50fps in RTX3070 in windows10 and 50-55fps in Ubuntu. So maybe it can improve your performance and specifications of your model, Hope it can be useful.

**Comment by vicentowang** [comment]
@hujb48  have you include real time 3d visualization at 50-55 fps ? I use yolov5s+hrnet got 26. fps, but when i use 3d visualization ,it down to 10 fps.

**Comment by hujb48** [comment]
> @hujb48 have you include real time 3d visualization at 50-55 fps ? I use yolov5s+hrnet got 26. fps, but when i use 3d visualization ,it down to 10 fps.

@vicentowang no, the speed which I have tested is just for 2d pose detection, however I haven't visualize the 3d result in real-time but have simulated to output the 3d coordinate instead of the 3D real-time rendering, and the speed of the whole process of the pre- trained model together with the 3d coordinate output in real-time is 25fps.

**Comment by gao123qiang** [comment]
@hujb48 
1, how to edit the mediapipe, i can't find the python code  in solutions, only c++，
2, if solved, hotw to edit the ediapipe points to 17 points the model needs?
how to solve it?

**Comment by hujb48** [comment]
> @hujb48 1, how to edit the mediapipe, i can't find the python code in solutions, only c++， 2, if solved, hotw to edit the ediapipe points to 17 points the model needs? how to solve it?

hi, here you might find the answer. 
1.  https://google.github.io/mediapipe/solutions/pose.html   - "Python Solution API"
2.  you could figure out the common point between coco/Human 3.6M and Mediapipe, and pick them up by the corresponding index. This can also refer https://google.github.io/mediapipe/solutions/pose.htm - "Pose Landmark Model (BlazePose GHUM 3D)"

**Comment by gao123qiang** [comment]
@hujb48 
i have see it in the past ,
but
1, there are some different parts between human3.6 and mediapipe,  like head, neck ,thorax, spine ,how to get these, shou we reTrain a new model in mediapipe?
2, how to solve many people in video, i notice mediapipe maybe detect the most prominent person,should we add deepSort ?
there is only c++ code in mediapipe tracking.
